yarn.lock

```diff
@@ -507,6 +507,13 @@
     "@aws-cdk/region-info" "1.106.1"
     constructs "^3.3.69"
 
+"@aws-cdk/aws-imagebuilder@1.106.1":
```

Comment:
Looks like the change to add the imagebuilder example didn't include an update to the `yarn.lock`